ITISFoundation/osparc-simcore
# 🐛 Read-only user cannot open the iframes

## What do these changes do?


The issue is because if a user doesn't have access to the project wallet the button is disabled as it didn't fall back to the users default/preferred wallet.

![2023-11-20 08 57 25](https://github.com/ITISFoundation/osparc-simcore/assets/903587/71cda548-1d28-4287-81be-79d07b78b4b7)


## Related issue/s

- Fixes ITISFoundation/osparc-issues#1183



## How to test

1. Create a user
2. From another user share a project with the new user and make the user read only for that project
3. As the new user try access the project

PASS: If the issue is fixed the user should be able to enter the project without issue
FAIL: The new user is unable to enter the project



## Dev Checklist


## DevOps Checklist
